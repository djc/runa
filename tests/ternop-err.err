ternop-err.rns [2.11]: unmatched types 'anyint', '$Str'
    a = 1 if True else 'a'
          ^
