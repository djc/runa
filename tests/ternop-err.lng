def main(name: str, args: array[str]) -> int:
	a = 1 if True else 'a'
