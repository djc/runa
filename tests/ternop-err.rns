def main(name: &str, args: &array[str]) -> i32:
	a = 1 if True else 'a'
	return 0
