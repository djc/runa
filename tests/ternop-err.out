tests/ternop-err.rns [2.8]: unmatched types 'anyint', '&str'
    a = 1 if True else 'a'
          ^
